pcdsdevices/valve.py

```diff
@@ -227,6 +227,34 @@ class VGC(VRC):
                     doc=('individual valve MPS state for debugging'))
 
 
+class VFS(Device):
+    """Class for Fast Shutter Valve."""
+    request_close = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal',
+                        doc='Request Fast Shutter to Close')
+    request_open = Cpt(EpicsSignalWithRBV, ':OPEN_SW', kind='normal',
+                       doc='Request Fast Shutter to Open')
+    reset_vacuum_fault = Cpt(EpicsSignalWithRBV, ':FAULT_RESET', kind='normal',
+                             doc=('Reset Fast Shutter Vacuum Faults: fast'
+                                  'sensor triggered, fast sensor turned off'))
+    override_status = Cpt(EpicsSignalWithRBV, ':VFS_OVRD_ON', kind='normal',
+                          doc='Epics Readback on Override mode')
+    override_force_open = Cpt(EpicsSignalWithRBV, ':VFS_FORCE_OPN',
+                              kind='normal',
+                              doc=('Epics Command to force open the valve in'
+                                   'override mode'))
+    gfs_trigger = Cpt(EpicsSignalRO, ':TRIG_RBV', kind='normal',
+                      doc='Guage Fast Sensor Input Trigger')
+
+    position_close = Cpt(EpicsSignalRO, ':IS_CLOSED_RBV', kind='normal',
+                         doc='Fast Shutter Closed Valve Position')
+    position_open = Cpt(EpicsSignalRO, ':IS_OPEN_RBV', kind='normal',
+                        doc='Fast Shutter Open Valve Position')
+    vfs_vac_fault_ok = Cpt(EpicsSignalRO, ':FAULT_OK_RBV', kind='normal',
+                           doc='Fast Shutter Vacuum Fault OK')
+    vfs_mps_ok = Cpt(EpicsSignalRO, ':FFO_OK_RBV', kind='normal',
+                     doc='Fast Shutter Fast Fault Output OK')
```

Comment:
This one is another judgement call: is it better to have the redundant `vfs_` in these two components, or not?

Reply:
The vfs has been taken out and the doc string has been revised to provide a clearer explanation.